socorro/processor/mozilla_transform_rules.py

```diff
@@ -391,89 +391,21 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         return True
 
 
-def setup_product_id_map(config, local_config, args_unused):
-    database_connection = local_config.database_class(local_config)
-    transaction = local_config.transaction_executor_class(
-        local_config,
-        database_connection
-    )
-    sql = (
-        "SELECT product_name, productid, rewrite FROM "
-        "product_productid_map WHERE rewrite IS TRUE"
-    )
-    product_mappings = transaction(
-        execute_query_fetchall,
-        sql
-    )
-    product_id_map = {}
-    for product_name, productid, rewrite in product_mappings:
-        product_id_map[productid] = {
-            'product_name': product_name,
-            'rewrite': rewrite
-        }
-    return product_id_map
-
-
 class ProductRewrite(Rule):
-    required_config = Namespace()
-    required_config.add_option(
-        'database_class',
-        doc="the class of the database",
-        default='socorro.external.postgresql.connection_context.'
-                'ConnectionContext',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
-    required_config.add_option(
-        'transaction_executor_class',
-        default="socorro.database.transaction_executor."
-                "TransactionExecutorWithInfiniteBackoff",
-        doc='a class that will manage transactions',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
-
-    required_config.add_aggregation(
-        'product_id_map',
-        setup_product_id_map
-    )
-
     def __init__(self, config):
         super(ProductRewrite, self).__init__(config)
-        self.product_id_map = setup_product_id_map(
-            config,
-            config,
-            None
-        )
+        self.product_map = {
+            "{aa3c5121-dab2-40e2-81ca-7ea25febc110}": "FennecAndroid",
+        }
 
     def version(self):
-        return '1.0'
+        return '1.1'
```

Comment:
This will be the first non-1.0 rule. I'm pretty sure the versions are ignored. I'm hard-pressed to see how they get used.

Reply:
There are a bunch of 2.0 rules: ESRVersionRewrite, PluginContentURL, PluginUserComment, etc. I didn't want to bump to 2 since I had changed the inner workings but not the expected output or input. I'll bump it right to 2.0